src/math/Vector3.js

```diff
@@ -320,15 +319,9 @@ Object.assign( Vector3.prototype, {
 
 	unproject: function () {
 
-		var matrix = new Matrix4();
+		return this.applyMatrix4( camera.projectionMatrixInverse ).applyMatrix4( camera.matrixWorld );
```

Comment:
It looks like `camera` is missing as a parameter in the method signature.

Reply:
Ugh... it worked in testing because `camera` is global... Thanks for noticing...

Reply:
Fixed.